src/Holiday/Brandenburg.php

```diff
@@ -0,0 +1,30 @@
+<?php
+/**
+ * This software is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License version 3 as published by the Free Software Foundation
+ *
+ * This library is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details.
+ *
+ * @copyright  Copyright (c) 2012 Mayflower GmbH (http://www.mayflower.de)
+ * @license    LGPL v3 (See LICENSE file)
+ */
+namespace Holiday;
+
+class Brandenburg extends Germany
+{
+    protected function getHolidays($year)
+    {
+        $timezone = $this->timezone;
+
+        $data   = parent::getHolidays($year);
+        $data[] = new Holiday("6.1." . $year, "Dreikönigstag", $timezone);
```

Comment:
According to http://feiertage.net/bundeslaender.php Brandenburg has no "Dreikönigstag", even https://de.wikipedia.org/wiki/Gesetzliche_Feiertage_in_Deutschland#.C3.9Cbersicht_aller_gesetzlichen_Feiertage agrees with that. What is the source of your implementation?